.eslintrc

```diff
@@ -140,7 +140,7 @@
       "before": false,
       "after": true
     }],
-    "space-after-keywords": 2,       // http://eslint.org/docs/rules/space-after-keywords
```

Comment:
Please do not modify our ESLint settings. They are properly set already.

If it isn't working properly for you, you most likely have some older copy of ESLint locally conflicting.